python/cudf/cudf/_lib/unaryops.pyx

```diff
@@ -166,3 +166,27 @@ def nans_to_nulls(py_col):
         )
 
     return mask
+
+
+def isNULL(col):
```

Comment:
Let's call these functions `is_null` and `is_not_null` instead of `isNull` and `isNotNull`